src/components/Search/SearchInput.tsx

```diff
@@ -82,6 +82,22 @@ type SearchInputListener = {
   onStateChange: (isOpen: boolean) => void;
 };
 
+function parseSuggestions(suggestions: SearchAutoSuggestionResultItem[] = []) {
+  return suggestions.map((suggestion) => {
+    const [allTitle, bundleTitle, abstract] = suggestion.term.split(AUTOSUGGEST_TERM_DELIMITER);
+    const url = new URL(suggestion.payload);
+    const pathname = url.pathname;
+    const item = {
```

Comment:
Is there specific reason to assign this to a const instead of directly constructing it when returning value?

Reply:
None apart from better readability.